src/api-reference/expense/quick-expense/v3.quick-expense.markdown

```diff
@@ -160,65 +150,57 @@ Name|Type|Format|Description
 `LocationSubdivision`	|	`string`	|		|	The state, province, or other country subdivision where the expense was incurred. This is used to determine the Location ID when the quick expense is converted into an expense entry. Format: ISO 3166-2:2007 country subdivision.
 `PaymentTypeCode`	|	`string`	|		|	This element specifies the method of payment for the expense. Format: CASHX = Cash, CPAID = Company Paid, or PENDC = Pending Card Transaction (default)
 `ReceiptImageID`	|	`string`	|		|	The unique identifier for the image. The ReceiptImageID is returned in the ID element of the Post Receipt Image API response.
-`SpendCategoryCode`	|	`string`	|		|	The spend category code for the quick expense. The available spend category codes are consistent across all Concur products. The values are used in Concur reporting. Format: One of the Code values in the [Spend Category Code List](/tools-support/reference/spend-category-codes.html). Note: COCRM (Company Car Mileage) and PRCAR (Personal Car Mileage) are not currently supported.
+`SpendCategoryCode`	|	`string`	|		|	The spend category code for the quick expense. The available spend category codes are consistent across all SAP Concur products. The values are used in SAP Concur reporting. Format: One of the Code values in the [Spend Category Code List](/tools-support/reference/spend-category-codes.html). Note: COCRM (Company Car Mileage) and PRCAR (Personal Car Mileage) are not currently supported.
 `TransactionAmount`	|	`Decimal`	|		|	**Required** The total amount of the expense in the original currency, with up to three decimal places. Example: 123.654
 `TransactionDate`	|	`DateTime`	|		|	**Required** The date the expense was incurred. Format: YYYY-MM-DD
 `VendorDescription`	|	`string`	|		|	The descriptive text for the vendor for the quick expense. This often matches the Merchant Name found in a credit card transaction. Max Length: 64
 
-
 ### Request URL
 
 ```
 https://www.concursolutions.com/api/v3.0/expense/quickexpenses/gWr7QiDXTuWSO8bsFmK87bnP%24sC8fZmg
 ```
 
-
-### JSON example of a successful response
+### JSON Example of a Successful Response
 ```
 no content
 ```
 
-## <a name="delete"></a>Delete a quick expense
+## <a name="delete"></a>Delete a Quick Expense
 
     DELETE  /api/v3.0/expense/quickexpenses/{id}
 
-
 ### Parameters
 
 Name|Type|Format|Description
 ---|---|---|---
 `id`	|	`string`	|	`path`	|	**Required** The ID of the quick expense to be deleted
 `user`	|	`string`	|	`query`	|	The login ID of the user. Optional. The user must have the Web Services Admin (Professional) or Can Administer (Standard) user role to use this parameter.
 
-
 ### Request URL
 ```
 https://www.concursolutions.com/api/v3.0/expense/quickexpenses/gWr7QiDXTuWSO8bsFmK87bnP%24sC8fZmg
 ```
 
-
-### JSON example of a successful response
+### JSON Example of a Successful Response
```

Comment:
Let's remove this one since it has no content

Reply:
Fixed, I thought I got them all.